src/MainWindow.vala

```diff
@@ -69,4 +70,29 @@ public class Installer.MainWindow : Gtk.Dialog {
             stack.set_visible_child_name ("progress-view");
         });
     }
+
+    private string get_os_release () {
```

Comment:
This method should be static as it doesn't access anything from the class instance.